s3/pkg/service/multipart.go

```diff
@@ -46,7 +46,7 @@ func (s *s3Service) ListBucketUploadRecords(ctx context.Context, in *pb.ListBuck
 		return err
```

Comment:
pls set the err to the out so that upper framework can detect it.

Reply:
@hopkings2008 thanks,  will do it